stable/signalfx-agent/Chart.yaml

```
@@ -2,7 +2,8 @@ apiVersion: v1
 description: The SignalFx Kubernetes agent
```

also need to add: DEPRECATED in the beginning
```suggestion
description: DEPRECATED The SignalFx Kubernetes agent
```